cdc/model/changefeed.go

```diff
@@ -146,6 +146,8 @@ type ChangeFeedInfo struct {
 	Error  *RunningError         `json:"error"`
 
 	CreatorVersion string `json:"creator-version"`
+	// Epoch is the epoch of a changefeed, changes on every restart.
```

Comment:
how about refine this comment, what an `epoch` stand for ?

Reply:
Is `epoch` similar to `term`?